container-images/tcib/base/os/tobiko/run_tobiko.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+TOBIKO_DIR=/var/lib/tobiko
+
+if [ ! -z ${USE_EXTERNAL_FILES} ]; then
+    mkdir -p $TOBIKO_DIR/.config/openstack
+    cp $TOBIKO_DIR/external_files/clouds.yaml $HOME/.config/openstack/
+fi
+
+pushd ${TOBIKO_DIR}
+
+export OS_CLOUD=default
+echo "export OS_CLOUD=default" > .bashrc
+
+mkdir -p .downloaded-images
+curl https://cloud-images.ubuntu.com/minimal/releases/jammy/release/ubuntu-22.04-minimal-cloudimg-amd64.img -o .downloaded-images/ubuntu-minimal
+
+git clone https://opendev.org/x/tobiko
+
+pushd tobiko
+echo '
+[DEFAULT]
+debug = true
+log_file = tobiko.log
+
+[ubuntu]
+interface_name = enp3s0
+image_file = /var/lib/tobiko/.downloaded-images/ubuntu-minimal
+
+[keystone]
+interface = public' > tobiko.conf
+
+python3 -m tox -e py3 --notest
+sleep 9999999
```

Comment:
Why this sleep here? You can also run tox -e py3,scenario ?

Reply:
Done.
I did something wrong when I removed the merge commit:
https://github.com/openstack-k8s-operators/tcib/compare/09091dbb266ec0f8b5afa68434a68f376c5b4f10..43e43d93c18255163da55fadb26bba6f31664881